### DISCUSSION TITLE
Two tariff LCD reading #2

### DISCUSSION THREAD

**Discussion by skaa** [jomjol/AI-on-the-edge-device#674] [Q&A] [answered]
Explanation to my previous question. The readings are displayed alternately in one place on the LCD screen, but with different markings. For example:
`45115,82 T1` - displayed for the first 5 seconds,
`18631,18 T2` - the next 5 seconds,
`63747,00 TA` - the next 5 seconds (sum of T1 and T2).
I need to take `T1` and `T2` readings.

**Comment by jomjol** [comment, marked as answer]
Okay thanks for the explanation. Unfortunately this is not possible as there is no trigger possible for controlling the image taking on an external status (like your T1 / T2 / TA information). Additionally the image processing takes about 30-90 seconds.
You could try to get the image just by change and hope, that over time you get all informations. Current there is no option to read the T1/T2/TA characters.

**Reply by RomchikL** [reply]
Are there any changes on this question? I have a similar electric metet

**Reply by jomjol** [reply]
No changes so far

**Comment by Franz4596** [comment]
My powermeter was changed into one with an alternating display (Zweirichtungszähler):
![PowerMeterAlternating](https://github.com/user-attachments/assets/7dc69109-826b-433d-b726-46e32ab04c3f)
Problem: the values are changing every 10s.  
Is there meanwhile a possibility to read both values?
Perhaps by taking an measurement intervall of 184s and with separate the values according "1.8.0" and "2.8.0" to get an full interval of 368s?

**Reply by SybexX** [reply]
The "WiFi IR Schreib-Lesekopf Tasmota" is more suitable for this.
